plugin/pkg/auth/authorizer/node/node_authorizer_test.go

```diff
@@ -521,17 +522,17 @@ func TestAuthorizerSharedResources(t *testing.T) {
 }
 
 type sampleDataOpts struct {
-	nodes int
-
-	namespaces int
-
+	nodes       int
+	namespaces  int
 	podsPerNode int
 
 	attachmentsPerNode int
 
-	sharedConfigMapsPerPod int
-	sharedSecretsPerPod    int
-	sharedPVCsPerPod       int
+	sharedConfigMapsPerNamespace int
```

Comment:
it's not immediately obvious what this controls... godoc? same for sharedSecretsPerNamespace

Reply:
done

Reply:
doc the behavior that `shared*PerPod` is used if greater than this number

Reply:
done